examples/avm/addressFromBuffer.ts

```diff
@@ -1,19 +1,36 @@
-import { Avalanche, Buffer } from "../../dist"
+import { Avalanche, Buffer, BinTools } from "../../dist"
 import { AVMAPI } from "../../dist/apis/avm"
 
-const ip: string = "localhost"
```

Comment:
Please revert the changes to the example scripts. This PR should only have changes for the `Fix generic object injection sink`